src/widgets/PrescriptionCart.js

```diff
@@ -25,7 +27,12 @@ import { recordKeyExtractor } from '../pages/dataTableUtilities';
  * @prop {Func}  onOptionSelection Callback when an option in the dropdown is selected.
  * @prop {Func}  onRemoveItem      Callback when this row is removed.
  */
-export const PrescriptionCart = ({ items, onChangeQuantity, onOptionSelection, onRemoveItem }) => {
+export const PrescriptionCartComponent = ({
```

Comment:
this component doesn't need to be exported

Reply:
#1821 so I can get these PR's off the books, hope that's ok!